src/clj/rems/api/schema.clj

```diff
@@ -65,6 +65,7 @@
   {:resource s/Str
    :application-id s/Num
    :start s/Str
+   :end s/Str
```

Comment:
the tests don't test `:end` at all. what value do we get when the db doesn't have an end date?

Reply:
I think I'll add some date checking here because this is a very valuable API anyway. Even though it may duplicate something of the db level tests.

Reply:
I'll add improved acceptance tests in another PR because we want this fix to be deployed for tomorrow.